src/elements/Stripe.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box } from '@chakra-ui/react'
+import { useParams } from 'react-router-dom'
+import { CheckoutForm, CheckoutReturn } from '~components/Faucet/Stripe'
+
+export const StripeCheckout = () => {
+  const { amount }: { amount?: string } = useParams()
+
+  return (
+    <Box>
+      <CheckoutForm amount={amount} />
+    </Box>
```

Comment:
These extra surrounding boxes look weird. Considering you're adding them on each view, and that these are routing elements, wouldn't make more sense to have such Box defined in the layout?